join.spec.md

```diff
@@ -187,9 +181,9 @@ Join a field with a particular subgraph.
 
 ```graphql definition
 directive @join__field(
-  graph: join__Graph!,
-  requires: ref__Fragment!,
-  provides: ref__Fragment!
+  graph: join__Graph
+  requires: String!
+  provides: String!
 ) on FIELD_DEFINITION
 ```
 
```

Comment:
On the next few lines, it shouldn't say `@join`, but also instead of using `@join` as a verb it should talk about a directive being on the type.

Reply:
Fixed